src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/ReconfigureFundingCycleForm.tsx

```diff
@@ -155,18 +163,35 @@ export function V2V3ReconfigureFundingCycleForm() {
           fundAccessConstraints={
             editingProjectData.editingFundAccessConstraints
           }
+          nftRewards={editingProjectData.editingNftRewards.rewardTiers}
         />
-
-        <Button
-          loading={reconfigureLoading}
-          onClick={reconfigureFundingCycle}
-          disabled={!fundingHasSavedChanges && !nftsWithFalseDataSourceForPay}
-          type="primary"
-        >
-          <span>
-            <Trans>Deploy funding cycle configuration</Trans>
-          </span>
-        </Button>
+        {nftDrawerHasSavedChanges ? (
+          <Space size="middle" direction="vertical">
+            <div style={{ display: 'flex' }}>
+              <h2 style={{ marginRight: 5 }}>1.</h2>
+              <SetNftOperatorPermissionsButton
+                onConfirmed={() => setNftOperatorConfirmed(true)}
+              />
```

Comment:
**suggestion** perhaps hide or disable this if the permission has already been granted?

Reply:
Done this now. Tested and it's working well 👍